help/policies/code_of_conduct_enforcement.md

```diff
@@ -0,0 +1,39 @@
+# arXiv Code of Conduct Enforcement Options
+
+arXiv is a platform where researchers engage with one another to participate in science, software development, and scholarly communications. It is also a community.
+
+Our valued community of authors, readers, moderators, advisory board members, staff, collaborators, and supporters expects and deserves a respectful environment, which is why we adhere to a [Code of Conduct](code_of_conduct), [moderation standards]( /help/moderation), and other policies. 
```

Comment:
Suggestion: change "advisory board members" to "advisors"

Reply:
accepted